test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -21,10 +21,14 @@ public void elementsAreUnique() throws Exception {
         assertAreUnique(1);
         assertAreUnique("");
         assertAreUnique("abc");
+        assertAreUnique(2.0);
 
         // all objects unique
         assertAreUnique("abc", "ab", "a");
         assertAreUnique(1, 2);
+        assertAreUnique(1, 2, null);
+        assertAreUnique("a", "b", null);
+        assertAreUnique("", null);
```

Comment:
Do consider cases where the null is not the last element. What about lists? Will nulls within a list return true or false?